tests/runner.js

```diff
@@ -57,10 +58,9 @@ function checkOnlyInTests() {
 function runMocha() {
   console.time('Mocha Tests Running Time');
   mocha.run(function(failures) {
-    process.on('exit', function() {
-      console.timeEnd('Mocha Tests Running Time');
-      process.exit(failures);
-    });
+    console.timeEnd('Mocha Tests Running Time');
+    exit.releaseExit();
```

Comment:
I don't believe this is correct. If exit was captured, we should allow it to naturally exit and allow pending work to flush, this disables that and then merely exists.